site/app/controllers/PollController.php

```diff
@@ -333,6 +344,16 @@ public function submitEdits() {
                 new RedirectResponse($this->core->buildCourseUrl(['polls']))
             );
         }
+        $file_path = null;
+        if (isset($_FILES['image_file']) && $_FILES["image_file"]["name"] !== "") {
+            $file = $_FILES["image_file"];
+            $current_file_path = $this->core->getQueries()->getPoll($_POST["poll_id"])->getImagePath();
+            if ($current_file_path != null) {
```

Comment:
```suggestion
            if ($current_file_path !== null) {
```

I really need to activate this lint rule. 🤔